ConanInstall.cmake

```diff
@@ -60,7 +60,7 @@ if(NOT CONAN_OPENSTUDIO_ALREADY_RUN)
   list(APPEND CONAN_BUILD "missing")
 
   if (BUILD_TESTING)
-    set(CONAN_GTEST "gtest/1.10.0#ef88ba8e54f5ffad7d706062d0731a40")
+    set(CONAN_GTEST "gtest/1.10.0#2210002307b82e555f7bec6c9ec6c839")
```

Comment:
The other revision doesn't build on Apple clang version 13.1.6 which become more stringent with warnings